From 75500b52feccd0bba9d1683ad39f29d99b0affae Mon Sep 17 00:00:00 2001
From: Tofe <chris.chapuis@gmail.com>
Date: Tue, 7 Jun 2022 15:37:39 +0200
Subject: [PATCH] LuneOS style: module name is now QtQuick.Controls.LuneOS

With Qt6 the style name is the name of the module, with the eventual prefixes. Also, Controls.2 has been removed, and the LuneOS style now lies in QtQuick/Controls subfolder.

---
 src/bus/client/NativeContainer.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/bus/client/NativeContainer.cpp b/src/bus/client/NativeContainer.cpp
index 8ac41ac..07af75a 100644
--- a/src/bus/client/NativeContainer.cpp
+++ b/src/bus/client/NativeContainer.cpp
@@ -195,7 +195,7 @@ void NativeContainer::launch(RunningAppPtr runningApp, LunaTaskPtr lunaTask)
 
     if (AppType::AppType_Native_Qml == type) {
         if(runningApp->getLaunchPoint()->getAppDesc()->useLuneOSStyle())
-            runningApp->getLinuxProcess().addEnv("QT_QUICK_CONTROLS_STYLE", "LuneOS");
+            runningApp->getLinuxProcess().addEnv("QT_QUICK_CONTROLS_STYLE", "QtQuick.Controls.LuneOS");
     }
 
     runningApp->setLS2Name(Logger::format("%s-%d", runningApp->getAppId().c_str(), s_instanceCounter));
